ste/mgr-JobPartMgr.go

```diff
@@ -877,6 +877,18 @@ func (jpm *jobPartMgr) Close() {
 	jpm.httpHeaders = common.ResourceHTTPHeaders{}
 	jpm.metadata = common.Metadata{}
 	jpm.preserveLastModifiedTime = false
+
+	/*
+	 * Set pipeline to nil, so that jpm/JobMgr can be GC'ed.
+	 *
+	 * TODO: We should not need to explicitly set this to nil but today we have a yet-unknown ref on pipeline which
```

Comment:
Please add all the TODOs from your PR into your backlog for future references.

Reply:
Added them as PBI for xdm